dynmotd

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
```

Comment:
my original idea was to provider a set of combinable different scripts.

this is why there was a directory and the script is named `00-basic`.

for example we could add in future, optional script to add to basic linux info like aws, azure ...

can we keep this ?

Reply:
Sure